man/systemctl.xml

```diff
@@ -567,6 +567,21 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
           <option>ExecStartPre=</option>, etc.) </para></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><command>mount-image</command> <replaceable>UNIT</replaceable> <replaceable>PATH</replaceable> [<replaceable>PATH</replaceable> [<replaceable>PARTITION_NAME</replaceable><replaceable>MOUNT_OPTIONS</replaceable>]]</term>
+
+          <listitem><para>Mounts an image from the host into the specified unit's view. The first path argument is the source
+          image on the host, the second path argument is the destination directory in the unit's view. Any following argument is
+          interpreted as a tuple of partition name and comma-separated list of mount options for that partition. The format is the
```

Comment:
MountImages= uses colons, which this stuff should too?

Reply:
Between partition and list of mount options? It felt simpler to type by just separating with spaces, eg:

```systemctl mount-image foo bar root a,b,c boot e,f,g```

Reply:
changed